### PR TITLE
🎨 adding optional override of dnsrr mode for testing purpose on master (⚠️ devops)

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
- Because of 🐛 dynamic sidecar is not able to resolve DNS of storage issue, this is an experiment to try to fix it (currently will be temporary turn on master) 
- documentation: https://docs.docker.com/engine/swarm/networking/#configure-service-discovery

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

## DevOps Checklist
- ⚠️ added new variable `DYNAMIC_SIDECAR_ENDPOINT_SPECS_MODE_DNSRR_ENABLED`
  - https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/423

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]

File: services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/sidecar.py [hunk 1]
```diff
@@ -97,6 +97,11 @@ def warn_if_any_values_provided(cls, value: dict) -> dict:
 
 
 class DynamicSidecarSettings(BaseCustomSettings, MixinLoggingSettings):
+    DYNAMIC_SIDECAR_ENDPOINT_SPECS_MODE_DNSRR: bool = Field(
+        default=False,
+        auto_default_from_env=True,
+        description="Override default endpoint spec mode 'vip' to 'dnsrr'",
```
Comment (GitHK) [inline services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/sidecar.py:103]:
```suggestion
        description="dynamic-sidecar's service "endpoint_spec" with {"Mode": "dnsrr"}",
```

**Review by GitHK — APPROVED** [review]
👍 please update the docstring

**Comment by YuryHrytsuk** [comment]
Does it require ENV changes?
image

**Comment by matusdrobuliak66** [comment]
> Does it require ENV changes? image

https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/423

**Review by YuryHrytsuk — APPROVED** [review]

**Pull request merged**